app/services/users/profile.rb

```diff
@@ -63,12 +64,21 @@ def account
       nil
     end
 
+    def demographics
```

Comment:
Note: Note the location of this call is here instead of user since we only need the demographics data when the user endpoint is called vs the user object is used in many other places.